src/frontend/src/flows/faq/index.ts

```diff
@@ -0,0 +1,70 @@
+import { html, render } from "lit-html";
+import { compatibilityChart } from "../../components/compatibilityChart";
+
+import { questions } from "./questions";
+
+// re-export for ease of use
+export { questions } from "./questions";
+
+// The FAQ page
+const pageContent = html`
+  <style>
+    /* briefly flash the question when redirected to a particular question */
+    @keyframes example {
+      0% {
+        background-color: transparent;
+      }
+      50% {
+        background-color: var(--grey-100);
+      }
+      100% {
+        background-color: transparent;
+      }
+    }
+
+    a {
+      color: var(--grey-800);
+    }
+
+    ul {
+      list-style-type: none;
+    }
+
+    li {
+      border-radius: 0.5rem;
+      padding: 0.5rem;
+    }
+    :target {
+      animation-name: example;
+      animation-duration: 3s;
+    }
+  </style>
+  <div class="container" id="faq">
+    <h1>FAQ</h1>
```

Comment:
```suggestion
    <h1>Frequently asked questions</h1>
```

Reply:
I've just looked at a few websites (Google, Twitter, Swiss Gov, Swiss railways, my bank, basically everything I could think of) and they all have "FAQ". Does that work for you?